lldb/tools/lldb-dap/DAP.h

```diff
@@ -202,6 +202,10 @@ struct DAP {
 
   lldb::SBFormat frame_format;
   lldb::SBFormat thread_format;
+
+  /// The value of stop-disassembly-display setting in LLDB.
+  std::string stop_disassembly_display;
```

Comment:
I'm wondering if we should be caching this. Querying the debugger for a setting should be cheap enough that we could do it whenever we need to know. The added benefit is that you'll be able to change the setting and have it take immediate effect, while with the current approach you'd have to create new debug session. 

Personally I'm leaning toward computing this once per request.